readme.md

```diff
@@ -243,6 +243,7 @@ Thanks for contributing! 🦋🙌
 - [](# "batch-mark-files-as-viewed") [Mark/unmark multiple files as “Viewed” in the PR Files tab. Click on the first checkbox you want to mark/unmark and then `shift`-click another one; all the files between the two checkboxes will be marked/unmarked as “Viewed”.](https://user-images.githubusercontent.com/1402241/79343285-854f2080-7f2e-11ea-8d4c-a9dc163be9be.gif)
 - [](# "first-published-tag-for-merged-pr") [Shows the first Git tag a merged PR was included in.](https://user-images.githubusercontent.com/16872793/81943321-38ac4300-95c9-11ea-8543-0f4858174e1e.png)
 - [](# "pr-jump-to-first-non-viewed-file") [Jumps to first non-viewed file in a pull request when clicking on the progress bar.](https://user-images.githubusercontent.com/16872793/85226580-3bf3d500-b3a6-11ea-8494-3d9b6280d033.gif)
+- [](# "highlight-deleted-and-added-files-in-diffs") [Shows whether files in PRs are being added or removed.](https://user-images.githubusercontent.com/16872793/89691560-3828fd00-d8d7-11ea-9107-ba7f9f4ec316.png)
```

Comment:
Can you use a more realistic screenshot with just a deleted file, added file, and some changed files? The arrows are fine

Reply:
I try later tonight

Reply:
Done